### DISCUSSION TITLE
How to install extensions via command line?

### DISCUSSION THREAD

**Discussion by pftnhr** [datenstrom/community#874] [Ask a question] [answered]
I was wondering, since I can't get to a computer at the moment, whether it is possible to install extensions via CLI, and if so, how. The background is that I have just updated a Yellow-powered website and the maintenance extension (not Giovanni's) did not come along and I would like to install Giovanni's now.

Thanks in advance
Robert

**Comment by schulle4u** [comment, marked as answer]
Do you only have shell access to the website? If the [recommended installation methods](https://github.com/annaesvensson/yellow-update) don't work, try downloading the extension with `wget`, `curl` or any other command line tool that can handle file downloads for you. Change into your `system/extensions` directory and download the appropriate zip file there. 

`cd system/extensions/ && wget -O yellow-maintenance-main.zip https://github.com/GiovanniSalmeri/yellow-maintenance/archive/main.zip`

You should run the above command as the webserver's user to avoid potential permission issues. Finally open your website in a browser as usual, yellow then will take care of the actual installation.

**Reply by pftnhr** [reply]
No, I can get to the files in other ways, but not last night. I had made an update and afterwards the maintenance extension caused a fatal error. Now I have installed @GiovanniSalmeri|s and still get an error. But I'll sort that out with Giovanni itself.

The command
`cd system/extensions/ && wget -O yellow-maintenance-main.zip https://github.com/GiovanniSalmeri/yellow-maintenance/archive/main.zip`
works perfectly!
